core/domain/fs_services.py

```diff
@@ -318,7 +339,15 @@ def save_original_and_compressed_versions_of_image(
             micro_image_filepath, micro_image_content, mimetype=mimetype)
 
 
-def save_classifier_data(exp_id, job_id, classifier_data_proto):
+# The argument classifier_data_proto can accept instances of
+# `TextClassifierFrozenModel` class. But since, we excluded
+# proto_files/ from the static type annotations. This argument
+# is annotated as general object type.
+def save_classifier_data(
```

Comment:
`TextClassifierFrozenModel` is defined in proto/files which is excluded in mypy checks (https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/scripts/run_mypy_checks.py#L32 )

Reply:
After excluding protofiles, anything related to protofiles is considered of type Any. But here I used general object ( type ), because the value accepted by `classifier_data_proto` is an instance of `TextClassifierFrozenModel`.

Reply:
TODO is added